lib/middleware/prototype-admin-routes.js

```diff
@@ -34,4 +34,20 @@ router.post('/password', (req, res) => {
   }
 });
 
+router.get('/reset', (req, res) => {
+
+  const backHref = req.get('Referrer') || '/'
```

Comment:
Probably need to find a way to sanitize the Referrer header to check it’s a valid local path, given that it’s set by the user/browser.